mod_reforged/hooks/config/character_names.nut

```diff
@@ -0,0 +1,105 @@
+
+::Const.Strings.CharacterNames.extend([
+    "Agnar",
+    "Alard",
+    "Alaric",
+    "Albert",
+    "Albterus",
+    "Aldwin",
+    "Ansgar",
+    "Badulf",
+    "Baldarich",
+    "Baldo",
+    "Baldovin",
+    "Baldwin",
+    "Baranor",
+    "Bertilo",
+    "Bjorn",
+    "Brando",
+    "Burchard",
+    "Carolus",
+    "Cenric",
+    "Conrad",
+    "Dagfinn",
+    "Dunstan",
+    "Eckhart",
+    "Edgar",
+    "Egino",
+    "Elof",
+    "Emelrich",
+    "Eswig",
+    "Faramund",
+    "Farvald",
+    "Filibert",
+    "Franco",
+    "Friduhelm",
+    "Gaufrid",
+    "Gautvin",
+    "Gerbold",
+    "Gilbert",
+    "Giselbert",
+    "Gismund",
+    "Godric",
+    "Grimm",
+    "Hadubert",
+    "Hagano",
+    "Harold",
+    "Hartwin",
+    "Heidrich",
+    "Helmfried",
+    "Helmo",
+    "Helmut",
+    "Herman",
+    "Kuno",
+    "Lambert",
+    "Landulf",
+    "Lanzo",
+    "Leofric",
+    "Leonard",
+    "Leudagar",
+    "Leuthar",
+    "Leutwin",
+    "Manno",
+    "Meino",
+    "Meinrad",
+    "Milo",
+    "Norbert",
+    "Odo",
+    "Olvir",
+    "Oswin",
+    "Pipin",
+    "Randulf",
+    "Regin",
+    "Reinald",
+    "Reinhard",
+    "Richard",
+    "Roland",
+    "Sigimar",
+    "Silvan",
+    "Siward",
+    "Svenn",
+    "Theo",
+    "Theodar",
+    "Thiemo",
+    "Valdemar",
+    "Veremund",
+    "Waldo",
+    "Waldomar",
+    "Walther",
+    "Widald",
+    "Wido",
+    "Wigand",
+    "Wighart",
+    "Wilfred",
+    "Wolfhart",
+    "Wulfric"
+]);
+
+// This will also void any custom names from other mods that execute before Reforged
+::Const.Strings.KnightNames = [];
```

Comment:
> These are not strong enough arguments in my opinion to completely overwrite the KnightNames array and therefore enforce incompatibility with mods loading before Reforged.
I don't see the point to even plan for knight-exclusive names at this point when they will be buried under 300 other names.
If I could I wouldn't even be using KnightNames and instead use a `function generateKnightName()` but that wouldn't be  backwards compatible

If those mods (We are talking about probably 0,1%) really want their custom unique KnightNames anyways which appear once every 300 spawns they can manipulate the load-order. 
Just like if a mod wanted to modify `onAnySkillUsed` for `thrust`, `slash`, `gash` etc.
They probably throw most of those new KnightNames into the CharacterNames array anyways.

They can even replace the KnightNames to go back to a small hand-picked collection of very exotic names.

Reply:
There is a difference between a mechanical change that happens due to our design intention of how a skill should behave vs. a cosmetic mod that changes e.g. the names generated for entities. The argument for `thrust`, `slash`, `gash` etc. is not analogous here because the design of those skills is connected to the balance of the mod. Nevertheless, those changes to those skills are currently temporary and we can/should implement a more compatible system.

In my opinion, enforced breaking of compatibility for such a small cosmetic thing, based on the arguments you gave i.e. (a) a few names feel weird to you and (b) you don't want to deal with two separate arrays, is both unnecessary and unwise.